plasma_framework/docs/contracts/PaymentStandardExitRouter.md

```diff
@@ -66,20 +66,21 @@ function (PlasmaFramework plasmaFramework, uint256 ethVaultId, uint256 erc20Vaul
 | txFinalizationVerifier | ITxFinalizationVerifier |  | 
 | safeGasStipend | uint256 |  | 
 
-### standardExits
+### standardexits
```

Comment:
Ah...I guess you did this manually?
Can you run `npn run docgen` instead (under `plasma_framework/` folder)?